README.md

```diff
@@ -1,7 +1,6 @@
 django-zen-queries
 ====================
 
-[![Build Status](https://travis-ci.com/dabapps/django-zen-queries.svg?branch=master)](https://travis-ci.com/dabapps/django-zen-queries)
```

Comment:
Can we replace this with the GitHub actions badge `![CI](https://github.com/dabapps/django-zen-queries/workflows/CI/badge.svg)`